src/components/instantlaunches/launch/index.js

```diff
@@ -84,13 +84,7 @@ const InstantLaunchStandalone = (props) => {
     if (isLoading) {
         return <LoadingAnimation />;
     } else if (error || resourceError) {
-        const err = error || resourceError;
-        return (
-            <ErrorTypographyWithDialog
-                errorMessage={t("instantLaunchError")}
-                errorObject={err}
-            />
-        );
+        return <WrappedErrorHandler errorObject={error || resourceError} />;
```

Comment:
This should also have a `baseId`, though I don't see one already setup for this button wrapper 🤔 
I guess `ids.BASE` should work for now 🤷

Reply:
Yeah, I haven't been sure what to do for the debug IDs. I figure we don't really use them anymore really anyway, but I'll add... something, at least.